apps/web/src/modules/create-dao/components/Artwork/ArtworkForm.tsx

```diff
@@ -49,73 +50,81 @@ export const Artwork: React.FC<ArtworkProps> = ({ title }) => {
       validationSchema={validationSchemaArtwork}
       onSubmit={handleSubmit}
     >
-      {(formik) => (
-        <Form>
-          <TextArea
-            {...formik.getFieldProps('projectDescription')}
-            inputLabel={'Collection Description'}
-            formik={formik}
-            id={'projectDescription'}
-            onChange={formik.handleChange}
-            onBlur={formik.handleBlur}
-            errorMessage={
-              formik.touched?.projectDescription && formik.errors?.projectDescription
-                ? formik.errors?.projectDescription
-                : undefined
-            }
-            placeholder={'Nouns is an experiment which combines...'}
-          />
-
-          <ArtworkUpload
-            {...formik.getFieldProps('artwork')}
-            inputLabel={'Artwork'}
-            formik={formik}
-            id={'artwork'}
-            onChange={formik.handleChange}
-            onBlur={formik.handleBlur}
-            helperText={
-              'Builder uses folder hierarchy to organize your assets. Upload a single folder containing a subfolder for each trait. Each subfolder should contain every variant for that trait.'
-            }
-            errorMessage={
-              formik.touched.artwork && formik.errors?.artwork
-                ? formik.errors?.artwork
-                : undefined
-            }
-          />
+      {(formik) => {
+        return (
+          <Form>
+            <TextArea
+              {...formik.getFieldProps('projectDescription')}
+              inputLabel={'Collection Description'}
+              formik={formik}
+              id={'projectDescription'}
+              onChange={(e) => {
+                formik.handleChange(e)
```

Comment:
perhaps we can make this a separate function just for readability -- this also exposes the same issue exists on `ArtworkUpload` -- can we also make this change there.

Reply:
For sure!